VSTSC : error TS6049: Build: Unsupported locale 'sv-SE'.

TypeScript Version: 1.8.29.0 
Microsoft Visual Studio Professional 2015 - Version: 14.0.25123.00 Update 2
Microsoft .NET Framework Version: 4.6.01055
_Se bottom of message for complete version info from Visual Studio => Help => About_


All of a sudden I am getting this error when I build:
`VSTSC : error TS6049: Build: Unsupported locale 'sv-SE'.`

The command that precides the error message above is:
`C:\Program Files (x86)\Microsoft SDKs\TypeScript\1.8\tsc.exe  --sourcemap --target ES5 --noEmitOnError --locale sv-SE  "file1.ts" "file2.ts" "file3.ts" ... <Many more ts-files> ... "fileX.ts"`


Here is the part of the log where typescrip is mentioned
`8>Target "PreComputeCompileTypeScript" in file "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\TypeScript\Microsoft.TypeScript.targets" from project "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\MyApplicationWebMobileApplication.csproj" (target "CompileTypeScript" depends on it):
8>Using "VsTsc" task from assembly "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\TypeScript\TypeScript.tasks.dll".
8>Task "VsTsc"
8>  C:\Program Files (x86)\Microsoft SDKs\TypeScript\1.8\tsc.exe  --sourcemap --target ES5 --noEmitOnError --locale sv-SE "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\MyApplication.Mobile.Common.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\MyApplication.Mobile.LocalBlup.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\typings\jquerymobile\jquerymobile.d.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\typings\jquery\jquery.d.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\typings\modernizr\modernizr.d.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\Views\Blup\Details.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\Views\Blup\Lamning.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\Views\Blup\SearchBlup.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\Views\Shared\AnteckningPlugin.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\Views\Home\Index.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\Views\Shared\CommonPopups.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\Views\Shared\BlupPopups.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\Views\Vagning\Vagning.ts"
8>Done executing task "VsTsc".
8>Done building target "PreComputeCompileTypeScript" in project "MyApplicationWebMobileApplication.csproj".
8>Target "PreComputeCompileTypeScriptWithTSConfig" skipped, due to false condition; ('@(ConfigFiles)' != '') was evaluated as ('' != '').
8>Target "CompileTypeScript" in file "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\TypeScript\Microsoft.TypeScript.targets" from project "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\MyApplicationWebMobileApplication.csproj" (target "Compile" depends on it):
8>Building target "CompileTypeScript" completely.
8>Output file "Scripts\MyApplication.Mobile.Common.js" does not exist.
8>Task "VsTsc"
8>  C:\Program Files (x86)\Microsoft SDKs\TypeScript\1.8\tsc.exe  --sourcemap --target ES5 --noEmitOnError --locale sv-SE "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\MyApplication.Mobile.Common.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\MyApplication.Mobile.LocalBlup.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\typings\jquerymobile\jquerymobile.d.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\typings\jquery\jquery.d.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\typings\modernizr\modernizr.d.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\Views\Blup\Details.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\Views\Blup\Lamning.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\Views\Blup\SearchBlup.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\Views\Shared\AnteckningPlugin.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\Views\Home\Index.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\Views\Shared\CommonPopups.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\Views\Shared\BlupPopups.ts" "C:\Code\Pri\MyApplication\Development\AzureWebApp\Source\MyApplicationWebMobileApplication\Scripts\Views\Vagning\Vagning.ts"
8>VSTSC : error TS6049: Build: Unsupported locale 'sv-SE'.
8>  
8>  The command exited with code 1.
8>Done executing task "VsTsc" -- FAILED.
8>Done building target "CompileTypeScript" in project "MyApplicationWebMobileApplication.csproj" -- FAILED.`


### Versions
Microsoft Visual Studio Professional 2015
Version 14.0.25123.00 Update 2
Microsoft .NET Framework
Version 4.6.01055

Installed Version: Professional

LightSwitch for Visual Studio 2015   00322-40000-00000-AA915
Microsoft LightSwitch for Visual Studio 2015

Windows Phone SDK 8.0 - ENU   00322-40000-00000-AA915
Windows Phone SDK 8.0 - ENU

Visual Basic 2015   00322-40000-00000-AA915
Microsoft Visual Basic 2015

Visual C# 2015   00322-40000-00000-AA915
Microsoft Visual C# 2015

Visual C++ 2015   00322-40000-00000-AA915
Microsoft Visual C++ 2015

Visual F# 2015   00322-40000-00000-AA915
Microsoft Visual F# 2015

Application Insights Tools for Visual Studio Package   5.2.60328.3
Application Insights Tools for Visual Studio

ASP.NET and Web Tools 2015 (RC1 Update 1)   14.1.20203.0
ASP.NET and Web Tools 2015 (RC1 Update 1)

ASP.NET Web Frameworks and Tools 2012.2   4.1.41102.0
For additional information, visit http://go.microsoft.com/fwlink/?LinkID=309563

ASP.NET Web Frameworks and Tools 2013   5.2.40314.0
For additional information, visit http://www.asp.net/

Azure App Service Tools v2.9   14.0.20316.0
Azure App Service Tools v2.9

Azure Data Lake Node   1.0
This package contains the Data Lake integration nodes for Server Explorer.

Azure Data Lake Tools for Visual Studio   2.0.6000.0
Microsoft Azure Data Lake Tools for Visual Studio

Clang with Microsoft CodeGen   14.0.25115
Clang with Microsoft CodeGen

Common Azure Tools   1.7
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

DataFactoryProject   1.0
Microsoft Data Factory Package

GitHub.VisualStudio   1.0
A Visual Studio Extension that brings the GitHub Flow into Visual Studio.

JavaScript Language Service   2.0
JavaScript Language Service

JavaScript Project System   2.0
JavaScript Project System

KofePackagePackage Extension   1.0
KofePackagePackage Visual Studio Extension Detailed Info

Microsoft Azure Data Factory Node Node   1.0
Azure Data Factory extension for Visual Studio Server Explorer.

Microsoft Azure HDInsight HQL Service   2.0.6000.0
Language service for Hive query

Microsoft Azure HDInsight Tools for Visual Studio   2.0.6000.0
An integrated development environment for HDInsight application development.

Microsoft Azure Mobile Services Tools   1.4
Microsoft Azure Mobile Services Tools

Microsoft Azure Tools   2.8
Microsoft Azure Tools for Microsoft Visual Studio 2015 - v2.8.40211.2

Microsoft Azure Tools   2.9
Microsoft Azure Tools for Microsoft Visual Studio 2015 - v2.9.40323.3

Microsoft MI-Based Debugger   1.0
Provides support for connecting Visual Studio to MI compatible debuggers

Microsoft Team Foundation Server 2015 Power Tools   14.0
Power Tools that extend the Team Foundation Server integration with Visual Studio.

Microsoft Visual Studio Process Editor   1.0
Process Editor for Microsoft Visual Studio Team Foundation Server

NuGet Package Manager   3.4.2
NuGet Package Manager in Visual Studio. For more information about NuGet, visit http://docs.nuget.org/.

Office Developer Tools for Visual Studio 2015 ENU   14.0.23025
Microsoft Office Developer Tools for Visual Studio 2015 ENU

PreEmptive Analytics Visualizer   1.2
Microsoft Visual Studio extension to visualize aggregated summaries from the PreEmptive Analytics product.

Python Tools for Visual Studio   2.2.40315.00
Python Tools for Visual Studio provides IntelliSense, projects, templates, Interactive windows, and other support for Python developers.

Python Tools for Visual Studio - Django Integration   2.2.40315.00
Provides templates and integration for the Django web framework.

Python Tools for Visual Studio - Profiling Support   2.2.40315.00
Profiling support for Python projects.

SQL Server Data Tools   14.0.60311.1
Microsoft SQL Server Data Tools

TFS Auto Shelve   5.3
Create a shelveset for the latest version of all pending changes, automatically.  Pending changes from the local TFS workspaces will be shelved to the TFS Server every "x" number of minutes when solutions are opened.

ToolWindowHostedEditor   1.0
Hosting json editor into a tool window

TypeScript   1.8.29.0
TypeScript tools for Visual Studio

Visual C++ for Cross Platform Mobile Development (Android)   14.0.25116.00
Visual C++ for Cross Platform Mobile Development (Android)

Visual C++ for Cross Platform Mobile Development (iOS)   14.0.25116.00
Visual C++ for Cross Platform Mobile Development (iOS)

Visual Studio Spell Checker   VSSpellChecker
An editor extension that checks the spelling of comments, strings, and plain text as you type.
https://GitHub.com/EWSoftware/VSSpellChecker

Visual Studio Tools for Apache Cordova   Update 8
Visual Studio Tools for Apache Cordova

Visual Studio Tools for Universal Windows Apps   14.0.25123.00
The Visual Studio Tools for Universal Windows apps allow you to build a single universal app experience that can reach every device running Windows 10: phone, tablet, PC, and more. It includes the Microsoft Windows 10 Software Development Kit.

Workflow Manager Tools 1.0   1.0
This package contains the necessary Visual Studio integration components for Workflow Manager.

Xamarin   3.11.1537.0 (a11147b)
Visual Studio extension to enable development for Xamarin.iOS and Xamarin.Android.

Xamarin.Android   5.1.7.12 (53fce3730830417896a42f365a5ba35f1ee58d9d)
Visual Studio plugin to enable development for Xamarin.Android.

Xamarin.Forms Intellisense   1.0
Provides intellisense for Xamarin.Forms in the XML editor.

Xamarin.iOS   9.0.1.0 (73db6324ad6864caae30bf4010c185598d262e69)
Visual Studio extension to enable development for Xamarin.iOS.

Xamarin.iOS Unified Migration   1.0
Automated migration for Xamarin iOS Classic projects to Unified

Xamarin.TestCloud.Integration   1.0
Early preview of Xamarin Test Cloud integration

--- a/src/compiler/tsc.ts
+++ b/src/compiler/tsc.ts
@@ -46,7 +46,7 @@ namespace ts {
         // First try the entire locale, then fall back to just language if that's all we have.
         // Either ways do not fail, and fallback to the English diagnostic strings.
         if (!trySetLanguageAndTerritory(language, territory, errors)) {
-            trySetLanguageAndTerritory(language, undefined, errors)
+            trySetLanguageAndTerritory(language, undefined, errors);
         }
 
         return true;

--- a/src/compiler/tsc.ts
+++ b/src/compiler/tsc.ts
@@ -44,11 +44,9 @@ namespace ts {
         const territory = matchResult[3];
 
         // First try the entire locale, then fall back to just language if that's all we have.
-        if (!trySetLanguageAndTerritory(language, territory, errors) &&
-            !trySetLanguageAndTerritory(language, undefined, errors)) {
-
-            errors.push(createCompilerDiagnostic(Diagnostics.Unsupported_locale_0, locale));
-            return false;
+        // Either ways do not fail, and fallback to the English diagnostic strings.
+        if (!trySetLanguageAndTerritory(language, territory, errors)) {
+            trySetLanguageAndTerritory(language, undefined, errors)
         }
 
         return true;

